app/models/ems_cluster.rb

```diff
@@ -190,7 +190,7 @@ def remove_all_resource_pools
 
   # All RPs under this Cluster and all child RPs
   def all_resource_pools
-    descendants(:of_type => 'ResourcePool')[1..-1].sort_by { |r| r.name.downcase }
+    descendants(:of_type => 'ResourcePool').select { |r| !r.is_default }.sort_by { |r| r.name.downcase }
```

Comment:
@agrare I'm good with this change, can you modify the commit to link to the ancestry issue with a comment that the default resource_pool is NOT always the first item so we can't just drop the first one.. we have to check if it's the default.  Maybe even put a comment here because it's not intuitive if you see the default rp as the first item 99% of the time.